pom.xml

```diff
@@ -50,6 +50,7 @@
 
         <diffutils.version>1.3.0</diffutils.version>
 
+        <ehcache.version>2.10.4</ehcache.version>
         <ezvcard.version>0.11.3</ezvcard.version>
```

Comment:
In whois we already have a caching mechanism from spring I think. I sit possible to use that instead of adding a new dependency?

Reply:
Do you mean a different library that implements `org.springframework.cache.annotation.Cacheable` ?

Reply:
Please clarify @maggarwal13

Reply:
Sorry I got confused between whois and whois-internal. We already have this in whois-internal. Thanks for clarifying